e2e/test-retries/__tests__/logTestErrorsBeforeRetries.test.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+'use strict';
+
+let i = 0;
+jest.retryTimes(3);
+jest.setLogTestErrorsBeforeRetry(true);
```

Comment:
```suggestion
jest.retryTimes(3, {logErrorsBeforeRetry: true});=
```

I'd rather we just add an options bag to the existent call than adding a new method entirely

Reply:
Makes sense, I've updated it. Do we want to handle the Symbol differently as well? Right now, there is still an additional symbol for the `logErrorsBeforeRetry`.